cartographer_ros_msgs/CMakeLists.txt

```diff
@@ -24,24 +24,18 @@ set(PACKAGE_DEPENDENCIES
 find_package(catkin REQUIRED COMPONENTS message_generation ${PACKAGE_DEPENDENCIES})
 
 add_message_files(
-  FILES
-    LandmarkEntry.msg
-    LandmarkList.msg
-    StatusCode.msg
-    StatusResponse.msg
-    SubmapList.msg
-    SubmapEntry.msg
-    SubmapTexture.msg
-    SensorTopics.msg
-    TrajectoryOptions.msg
+  DIRECTORY
+    msg
```

Comment:
Does cmake still know when to reconfigure?

Reply:
"Works on my machine"